server/src/domain/search/search.service.spec.ts

```diff
@@ -84,15 +85,19 @@ describe(SearchService.name, () => {
   });
 
   describe(`init`, () => {
-    // it('should skip when search is disabled', async () => {
-    //   await sut.init();
+    it('should skip when search is disabled', async () => {
+      // since there is no repository to mock, because the feature flag SEARCH is just evaluated by env, it is necessary to spy on config core
```

Comment:
Take a look at `system-config.service.spec.ts`. You can set `process.env.TYPESENSE_ENABLED=false` before calling the method.

Reply:
I pushed changes with it working as expected. `init()` is called in the `beforeEach`, so you need to reset the mocks as well.

Reply:
Thank you :)